code/index.html

```diff
@@ -2,21 +2,265 @@
+    <link rel="manifest" href="/site.webmanifest">
     <!-- dont forget the meta tag for responsiveness -->
   </head>
+
   <body>
+
+    <!-- HEADER -->
+
     <header>
       <!-- Some kind of logo and navbar -->
+      <nav>
+        <div>
+          <img class="animate__animated animate__swing" src="./logo.svg" alt="">
+        </div>
+        <div>
+          <h1>Front End Ninja 🥷🏻</h1>
+        </div>
+        <div class="hamburger">
+          <div></div>
+          <div></div>
+          <div></div>
+        </div>
+
+        <div>
+          <a href="/html/">Home</a>
+          <a href="/css/">Search</a>
+          <a href="/js/">About</a>
+        </div>
+      </nav>
     </header>
 
     <section class="big-news">
       <!-- The big news content -->
+      <div class="big-news-container">
+        <a href="https://developer.mozilla.org/en-US/docs/Web/CSS/:hover">
+          <div>
+            <img src="https://images.unsplash.com/photo-1457131760772-7017c6180f05?ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=2072&q=80https://images.unsplash.com/photo-1457131760772-7017c6180f05?ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=2072&q=80" alt="">
+          </div>
+          <div>
+            <h3>How to :hover</h3>
+            <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. 
+              Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. 
+              Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. 
+              Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+          </div>
+        </a>
+      </div>
+      
+      <div class="big-news-container">
+        <a href="https://www.technigo.io">
+          <div>
+            <img src="https://images.unsplash.com/photo-1543616991-75a2c125ff5b?ixlib=rb-1.2.1&ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&auto=format&fit=crop&w=2100&q=80" alt="">
+          </div>
+
+          <div>
+              <h3>Find Your Passion</h3>
+              <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. 
+              Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. 
+              Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. 
+              Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+            
+          </div>
+        </a>
+      </div>
     </section>
 
-    <section class="">
+    <section class="small-news">
       <!-- The other news  -->
+      <div class="small-news-container">
```

Comment:
`small-news-container` almost sounds like it would "contain" `small-news` instead of vice-versa